composer.json

```diff
@@ -5,7 +5,8 @@
     "homepage": "https://spryker.com",
     "license": "MIT",
     "require": {
-        "php": ">=7.4",
+        "mikey179/vfsstream": "^1.6.7",
```

Comment:
why are we adding this to require?

Reply:
I dont think this is a valid change

Reply:
Where in /src/ do you see this dependency being needed or used?
This is for testing harness, so it by design, cannot be a require dependency.

Reply:
Yes. You are right. I moved this dependency to dev section and all good